pkg/trainer/replicas.go

```diff
@@ -499,3 +501,30 @@ func (s *TFReplicaSet) genPodName(index int32) string {
 func (s *TFReplicaSet) defaultPSConfigMapName() string {
 	return fmt.Sprintf("cm-ps-%v", s.Job.job.Spec.RuntimeId)
 }
+
+// SyncPdb will create a PDB for gang scheduling by kube-arbitrator.
+func (s *TFReplicaSet) SyncPdb() error {
+	minAvailable := intstr.FromInt(int(*s.Spec.Replicas))
+	pdb := &v1beta1.PodDisruptionBudget{
+		Spec: v1beta1.PodDisruptionBudgetSpec{
+			MinAvailable: &minAvailable,
+			Selector: &meta_v1.LabelSelector{
+				MatchLabels: s.Labels(),
```

Comment:
I think we can not use the TFReplicaSet's labels directly since TFJob's TFReplicaSets have different labels. We need to make sure that all the replicas of the TFJob are gang scheduled by the scheduler.

Reply:
Thanks, probably just specifying `tf_job_name` would be suitable. I'll fix it.